library/azure_rm_postgresqlserver.py

```diff
@@ -56,6 +56,22 @@
         description:
             - The maximum storage allowed for a server.
         type: int
+    geo_redundant_backup:
+        description:
+            - Choose between locally redundant(default) or geo-redundant backup. This cannot be updated after first deployment
+        type: bool
+        version_added: "2.9"
+    backup_retention_days:
+        description:
+            - Backup retention period between 7 and 35 days. 7 days by default if not set
+        type: int
+        version_added: "2.9"
+    storage_autogrow:
+        description:
+                - The storage automatically grows when close to be full.
+        type: bool
+        default: False
```

Comment:
For this default value. If user first set it to true. and then left it empty. User just want to keep it unchanged. However, it will automatically set this to false again due to the default value.

Reply:
You are right. Fixed